data/ai/micro_ais/cas/ca_fast_attack_utils.lua

```diff
@@ -184,7 +191,7 @@ function ca_fast_attack_utils.damage_rating_unit(attacker_info, defender_info, a
     -- Input parameters:
     --  @attacker_info, @defender_info: unit_info tables produced by ca_fast_attack_utils.get_unit_info()
     --  @att_stat, @def_stat: attack statistics for the two units
-    --  @is_village: whether the hex from which the attacker attacks is a village
+    --  @healing: the healing given by a village
```

Comment:
Should this actually be the healing given by the hex the attacker is on?

Reply:
Yeah, that's really a better description of what the code does.  I'll correct the comment.

I guess the exception would be if there are hexes on the map that provide healing but that don't count as villages. Does something like that exist (other than hexes next to healers)? I don't actually know how `wesnoth.get_villages()` works internally. But either way, given that this is supposed to be the _fast_ MAI, I would rather not add additional complexity to it.